sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/DeploymentController.java

```diff
@@ -25,7 +24,7 @@
 @RequestMapping("/deployments")
 class DeploymentController {
 
-  private static final DeploymentApi API = new DeploymentApi(getClient());
+  private static final DeploymentApi API = new DeploymentApi(new AiCoreService().client());
```

Comment:
```suggestion
  private static final DeploymentApi API = new DeploymentApi(new AiCoreService().getClient());
```

How about classic getters for `client()` and `destination()`. Especially if `Client` is not part of the class name of `AiCoreService`

Reply:
* _Power-point readiness_ dictates short API
* Omitting `get` prefix for immutable classes is (modern) common practice, see [Java record classes](https://docs.oracle.com/en/java/javase/17/language/records.html) for example